catalog/plugin/catalog-plugin-query-monitor-plugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,38 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version. 
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+>
+
+    <!--
+    NOTE: ConnectedSources are explicitly not monitored because their sourceId overlaps
+    with the local catalog provider. 
+    -->
+<!--    <service interface="ddf.catalog.querymonitor.QueryMonitorPlugin">
+        <service-properties>
+            <entry key="id" value="rootclass"/>
+        </service-properties>
+        <bean class="ddf.catalog.querymonitor.QueryMonitorPluginImpl"/>
+    </service>-->
+
+    <bean id="queryMonitor" class="ddf.catalog.querymonitor.QueryMonitorPluginImpl">
+        <!--init-method="init" destroy-method="destroy"> -->
+    </bean>
+
+    <service ref="queryMonitor" interface="ddf.catalog.plugin.PreFederatedQueryPlugin"/>
+    <service ref="queryMonitor" interface="ddf.catalog.plugin.PostFederatedQueryPlugin"/>
+    <service ref="queryMonitor" interface="ddf.catalog.querymonitor.QueryMonitorPlugin"/>
```

Comment:
You don't need to list this service multiple times, just do:
    <service ref="queryMonitor">
        <interfaces>
          <value>ddf.catalog.plugin.PreFederatedQueryPlugin</value>
          <value>ddf.catalog.plugin.PostFederatedQueryPlugin</value>
          <value>ddf.catalog.querymonitor.QueryMonitorPlugin</value>
        </interfaces>
    </service>